app/Livewire/Source/Github/Change.php

```
@@ -69,6 +97,52 @@ public function boot()
         }
     }
 
+    /**
+     * Sync data between component properties and model
+     *
+     * @param  bool  $toModel  If true, sync FROM properties TO model. If false, sync FROM model TO properties.
+     */
+    private function syncData(bool $toModel = false): void
```

PHPMD (2.15.0)

The method syncData has a boolean flag argument $toModel, which is a certain sign of a Single Responsibility Principle violation. (undefined)

(BooleanArgumentFlag)